content/en/docs/rate-limits.md

```diff
@@ -103,12 +104,17 @@ registered domain is `example.com`. In `new.blog.example.co.uk`, the registered
 domain is `example.co.uk`. We use the [Public Suffix
 List](https://publicsuffix.org/) to identify registered domains.
 
+If you're requesting a certificate for an IP address, this rate limit evaluates
+an IPv4 address as if it's a registered domain. For IPv6 addresses, it evaluates
+the /64 range that contains the address.
```

Comment:
This phrasing feels awkward to me for some reason. It definitely works, but I'm gonna try to brainstorm something different. Maybe:

> For IP addresses, we also try to treat the usual unit of sale (what you'd buy from your ISP or hosting provider) as the "registered domain". So for IPv4 addresses, we treat the exact address as the registered domain, and for IPv6 addresses, we treat the containing /64 range as the registered domain.

Does that sound better? I'm honestly not sure.

Reply:
I mashed these up a little. I wanted to keep the first clause to try to minimize anticipated confusion about the client's IP address connecting to the API (not this rate limit) vs. an IP address identifier in the certificate (yes this rate limit).